app/Database/Migrations/2024-08-10-144608_AddAccount.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+use CodeIgniter\Database\RawSql;
+
+class AddAccount extends Migration
+{
+    public function up()
+    {
+        $this->forge->addField([
+            'user_id' => [
+                'type' => 'INT',
+                'unsigned' => true,
+                'unique' => true,
+                'null' => false,
+            ],
+            'email' => [
+                'type' => 'VARCHAR',
+                'constraint' => 320,
+                'unique' => true,
+                'null' => false,
+            ],
+            'username' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'unique' => true,
+                'null' => false,
+            ],
+            'password' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => false,
+            ],
+            'password_change_required' => [
+                'type' => 'BOOLEAN',
+                'default' => false,
+                'null' => false,
+            ],
+            'created_at' => [
+                'type' => 'DATETIME',
+                'null' => false,
+            ],
+            'updated_at' => [
+                'type' => 'DATETIME',
+                'null' => false,
+            ],
+            'deleted_at' => [
+                'type' => 'DATETIME',
+                'null' => true,
+            ],
+        ]);
+        $this->forge->addKey('user_id', true);
+        $this->forge->addForeignKey('user_id', 'User', 'id', '', '');
```

Comment:
consider remove the last two arguments?

Reply:
Good catch, I didn't assume that those were default arguments. Will change this! 👍